architecture/additional_concepts/authentication.adoc

```diff
@@ -75,7 +75,9 @@ methods:
 
 OAuth Access Tokens::
 - Obtained from the {product-title} OAuth server using the `_<master>_/oauth/authorize` and `_<master>_/oauth/token` endpoints.
-- Sent as an `Authorization: Bearer...` header or an `access_token=...` query parameter
+- Sent as an `Authorization: Bearer...` header
+- Sent as an `access_token=...` query parameter for websocket requests prior to 3.6
+- Sent as a websocket subprotocol header in the form `base64url.bearer.authorization.k8s.io.<base64url-encoded-token>` for websocket requests in versions 3.6 and after
 
```

Comment:
I think may need to further conditionalize for Online and Dedicated. The added content may not be applicable to those products. Is that correct?